src/test/java/HandleTruthTest.java

```diff
@@ -1,8 +1,109 @@
+        expectedWordMaps.add("explain=1very=1guns=1saved=1lieutenant=1about=1your=1these=1saves=1would=1pick=1fathom=1grotesque=1incomprehensible=1because=1greater=1you=18give=1sleeps=1went=1lives=2in=3want=2loyalty=1myself=1them=1it=2then=1something=1as=2at=1knowing=1provide=2santiago=1curse=1weinberg=1entitled=1whos=1be=1probably=1world=1walls=2freedom=1responsibility=1are=1by=1have=5tragic=1backbone=1man=1stand=1inclination=1a=8think=1gonna=1words=2i=8suggest=1the=9places=1thank=1to=5under=1punchline=1use=3honor=1existence=1defending=1questions=1do=1manner=1while=2down=1that=6either=1son=1post=1than=1me=2talk=1has=1up=1those=1possibly=1which=1otherwise=1need=1like=1my=1parties=1know=1santiagos=1rises=1dont=3who=1deep=1code=1death=1rather=1for=1we=3weep=1life=1nor=1guarded=1can=1weapon=1not=1and=8men=1of=3said=1just=1live=1on=3spent=1way=2damn=1with=1what=2truth=1marines=1neither=1luxury=2time=1blanket=1wall=2");
+        expectedSortedMaps.add("18=[you]9=[the]8=[a, and, i]6=[that]5=[have, to]3=[in, use, of, dont, we, on]2=[lives, want, words, it, while, way, as, walls, what, provide, me, luxury, wall]1=[explain, very, guns, saved, lieutenant, about, your, these, saves, would, pick, fathom, grotesque, incomprehensible, because, greater, give, sleeps, went, loyalty, myself, them, then, something, at, knowing, santiago, curse, weinberg, entitled, whos, be, probably, world, freedom, responsibility, are, by, tragic, backbone, man, stand, inclination, think, gonna, suggest, places, thank, under, punchline, honor, existence, defending, questions, do, manner, down, either, son, post, than, talk, has, up, those, possibly, which, otherwise, need, like, my, parties, know, santiagos, rises, who, deep, code, death, rather, for, weep, life, nor, guarded, can, weapon, not, men, said, just, live, spent, damn, with, truth, marines, neither, time, blanket]");
+
+        //test numbers
+        testStrings.add("2 2 3 4 3 3 4 4 1 4");
+        expectedWordMaps.add("1=12=23=34=4");
+        expectedSortedMaps.add("4=[4]3=[3]2=[2]1=[1]");
+
+        for (int i = 0; i < testStrings.size(); i++) {
+            ArrayList<AbstractMap> mapList = HandleTruth.wordCount(testStrings.get(i));
```

Comment:
here you can actually just return the sortedMap Map<Integer, Set<String>> and use the equals function for comparison.

Reply:
Please see https://github.com/pdgetrf/you-cant-handle-the-truth/blob/solution/src/test/java/HandleTruthTest.java as an example.